### PR TITLE
Allow non-production builds

### PR DESCRIPTION
**Pull request by alexey-pelykh**
To allow
```
npm install -S ../tsoa
```

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
-*
 !dist/**
+typings
```
Comment (lukeautry) [inline .npmignore]: I don't think this should be necessary; typings directory shouldn't be there anyway.

**Comment by alexey-pelykh** [comment]
@lukeautry updated

**Comment by lukeautry** [comment]
@alexey-pelykh Looks like a little more got merged into this branch than you intended.

**Comment by alexey-pelykh** [comment]
@lukeautry indeed, sorry. Updated

**Pull request merged**